app/views/projects_close/closed.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
```

Comment:
I think in the past when we've used panels they've spanned the full page, but I don't think this is critical enough to hold this up. If it's a problem we can pick it up in snagging.

Reply:
I agree - but followed the prototype blindly for now! https://conversions-prototype.herokuapp.com/0-9/project-closed